README.md

```diff
@@ -51,38 +51,103 @@ dependencies:
 
 Run `pub get` to download and install the package.
 
-#### 2. Use the plugin in your application code to get the visitor identifier
+#### 2. Provide configuration to the plugin
 
 ```dart
 import 'package:fpjs_pro_plugin/fpjs_pro_plugin.dart';
 // ...
 
 // Initialization
 class _MyAppState extends State<MyApp> {
-  // ...
   @override
   void initState() async {
     super.initState();
     await FpjsProPlugin.initFpjs('<apiKey>'); // insert your actual API key here
   }
+  // ...
 }
+```
+
+You can also configure `region` and `endpoint` in `initFpjs` method, like below:
+```dart
+void doInit() async {
+  await FpjsProPlugin.initFpjs('<apiKey>');
+  await FpjsProPlugin.initFpjs('<apiKey>', endpoint: 'https://subdomain.domain.com');
+  await FpjsProPlugin.initFpjs('<apiKey>', region: 'eu');
+}
+```
+
+#### 3. Use the plugin in your application code to identify visitor
```

Comment:
Be consistent in punctuations, pls, there are headings with no period and with a period.